buildspec_integ.yml

```diff
@@ -9,7 +8,11 @@ phases:
       - mkdir -p /go/src/github.com/aws/amazon-ecs-cli
       - cp -R . /go/src/github.com/aws/amazon-ecs-cli/
       - cd /go/src/github.com/aws/amazon-ecs-cli/
+      - echo "installing dependencies..."
+      - go get github.com/wadey/gocovmerge
```

Comment:
why do we need to add coverage reporting to this stage? I'm not against it in principle but it will add time/dependencies  and we have it running in other build projects